hadoop-ozone/dist/src/main/smoketest/admincli/container.robot

```diff
@@ -96,5 +96,36 @@ Cannot close container without admin privilege
 Cannot create container without admin privilege
     Requires admin privilege    ozone admin container create
 
+Cannot reconcile container without admin privilege
+    Requires admin privilege    ozone admin container reconcile "${CONTAINER}"
+
 Reset user
     Run Keyword if      '${SECURITY_ENABLED}' == 'true'     Kinit test user     testuser     testuser.keytab
+
+Cannot reconcile open container
+    # At this point we should have an open Ratis Three container.
+    ${container} =      Execute          ozone admin container list --state OPEN | jq -r 'select(.replicationConfig.replicationFactor == "THREE") | .containerID' | head -n1
+    Execute and check rc    ozone admin container reconcile "${container}"    255
+    # The container should not yet have any replica checksums.
+    # TODO When the scanner is computing checksums automatically, this test may need to be updated.
+    ${data_checksum} =  Execute          ozone admin container info "${container}" --json | jq -r '.replicas[].dataChecksum' | head -n1
+    # 0 is the hex value of an empty checksum.
+    Should Be Equal As Strings    0    ${data_checksum}
+
+Close container
+    ${container} =      Execute          ozone admin container list --state OPEN | jq -r 'select(.replicationConfig.replicationFactor == "THREE") | .containerID' | head -1
+                        Execute          ozone admin container close "${container}"
+    ${output} =         Execute          ozone admin container info "${container}"
+                        Should contain   ${output}   CLOS
```

Comment:
```suggestion
                        Should contain   ${output}   CLOSED
```

Reply:
`CLOS` is used here (and in the existing `close container` test) because the state may either be `CLOSING` or `CLOSED` between when the container close command finishes and the container info command is executed. After this, we wait until the final state finishes moving to CLOSED. I can add a comment explaining this.